### DISCUSSION TITLE
What are the next release plans?

### DISCUSSION THREAD

**Discussion by henryju** [JetBrains/xodus#109] [General]
Hi,

There are a few fixes (like [XD-856](https://youtrack.jetbrains.com/issue/XD-856/Xodus-2.0.1-logs-an-exception-stack-trace-during-startup-when-running-under-JDK-17)) in master that have not been released. The last release is more than 1 year old, is there any plan to do a new one?

Thanks!

**Comment by andrii0lomakin** [comment]
Hi @henryju .
Sure, we already finished the new implementation (3.0 version, you can find it inside the branch with the same name) and currently intensively testing it on our environments.
Once we are sure that it is GA-ready we will release it. 
Though I can not provide you release date it is a question of a couple of months. 
We want to ensure users do not have issues with a new release.

**Reply by agermose** [reply]
Any news about next release?

**Reply by cpppwner** [reply]
Hi @andrii0lomakin,

out of curiosity, what are the next release plans?
According to https://www.jetbrains.com.cn/en-us/legal/third-party-software/?product=YTD, version 3.1 is already in use.

Thanks!

**Reply by andrii0lomakin** [reply]
@agermose @cpppwner We have added a statement to the project page that hopefully answers your questions.
Will be happy to see you among the users of our new database.
